app/models/course_user.rb

```diff
@@ -89,6 +89,8 @@ class CourseUser < ActiveRecord::Base
       order('achievement_count DESC, last_obtained_achievement ASC')
   end)
 
+  scope :order_alphabetically, ->(direction = :asc) { order(name: direction) }
```

Comment:
Yah we have both `order_by` and `ordered_by`, should standardise this. 

Use `ordered_by` then ? we have the discussion before that prefer using a noun as scope name.